test/validators/movie.test.js

```diff
@@ -1,7 +1,6 @@
 'use strict';
 
-const Joi = require('joi');
-
+const Joi            = require('joi');
```

Comment:
nit: don't need to align this because there's a newline and we typically separate external dependencies from internal files